### DISCUSSION TITLE
Search by fields

### DISCUSSION THREAD

**Discussion by antlio** [scalar/scalar#2844] [Ideas]
### Potential feature
To display fields among search request results.

### What happened
Searching for a specific field in a broad api documentation can be difficult.

>when the body has 70 fields and all of those have descriptions, etc., the page is extremely long and it's hard to find the field I am looking for. When I try to use the browser search functionality, it jumps to another request, which is quite annoying.
> I would like to be able to search only "Create new invoice" requests body fields.

![image](https://github.com/user-attachments/assets/e9686d57-2888-4854-a25e-bcb2075115a6)
![image](https://github.com/user-attachments/assets/6d567fcd-93c7-48c0-992f-9cf1c0811a52)

[posted by Rainmaker in discord](https://discord.com/channels/1135330207960678410/1135330208627564597/threads/1272486592019435611)

**Comment by hanspagel** [comment]
Oh yes, great feedback!
